src/Channel.cpp

```diff
@@ -106,6 +106,12 @@ void Channel::debanClient(Client& client) {
     removeClientFromMap(_blackList, client, "the user was not blacklisted");
 }
 
+bool Channel::isInChannel(Client& client) const {
```

Comment:
you can return the call to verifClientMap since it's a boolean for every function